jumpingman.py

```diff
@@ -0,0 +1,2 @@
+if (jumping)
```

Comment:
I hate this way of writing functions. Please fix!

Reply:
I won't resolve this. Because I think it's better to do it my way.